.github/workflows/intel.yml

```diff
@@ -112,6 +112,7 @@ jobs:
         set +e
         source /opt/intel/oneapi/setvars.sh
         set -e
+        export PATH=$PATH:/opt/intel/oneapi/compiler/2024.2/bin  # FIXME
         export CXX=$(which icpx)
         export CC=$(which icx)
```

Comment:
Exporting `PATH` should be unnecessary here, given that we execute `source /opt/intel/oneapi/setvars.sh` before. However, for some reason the Intel `setvars` script still adds the path to the 2025 version of the Intel compiler.